### DISCUSSION TITLE
How to use new PuLID v1.1 from Comfyui with SDXL model?

### DISCUSSION THREAD

**Discussion by xps2020** [ToTheBeginning/PuLID#165] [Q&A]
Anyone can tell the right way about
How to use new PuLID v1.1 from Comfyui with SDXL?

**Comment by gohan2091** [comment]
I would also like to know. I found a demo which I believe is the same? https://huggingface.co/spaces/yanze/PuLID it allows multiple input images which is exactly what I want in ComfyUI workflow.
